### DISCUSSION TITLE
Account deleted

### DISCUSSION THREAD

**Discussion by mouoent** [community/community#144432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I’m having a serious issue with my GitHub account (username: akarounis). It’s gone missing, whenever I try to visit my profile, I just get a "Page does not exist" message, even though I can still see the link in search results. This has really impacted my work, as I rely heavily on my GitHub account for my projects.

Here's what I've done so far:

1. I submitted a support ticket to GitHub about a week ago explaining the issue.

2. A few days ago, I followed up on the same ticket, hoping for a response.

3. Unfortunately, I haven’t heard back yet.

Has anyone else had this happen, or does anyone know how to get in touch with GitHub Support more directly? I really need to resolve this as soon as possible.

Thanks for any advice or help!

Best,
akarounis

**Comment by mastertanzeel** [comment]
Hi akarounis,

You might be making a typo (wrong username spelling). Please cross-check and lemme know if it helps to fix your error. You can login to your github account to check your username.
Thanks

**Reply by mouoent** [reply]
Hello!
No typos here, the account I am reffering to has the above mentioned username. I created my current account just to communicate with github support. Like I said, the account no longer exists, so I am unable to login. Here's a screenshot from one of the repositories I was a collaborator for proof. My name still appears there:

![tmp_d49de136-6922-4d66-b8db-a347bcbc7e11](https://github.com/user-attachments/assets/3f8c731f-fccb-48ce-add4-b44ed4efbbd4)

**Reply by mastertanzeel** [reply]
> Hello!
> No typos here, the account I am reffering to has the above mentioned username. I created my current account just to communicate with github support. Here's a screenshot from one of the repositories I was a collaborator for proof:
> 
> ![tmp_d49de136-6922-4d66-b8db-a347bcbc7e11](https://github.com/user-attachments/assets/3f8c731f-fccb-48ce-add4-b44ed4efbbd4)
> 

Have you ever changed your username?

**Reply by mouoent** [reply]
> > Hello!
> > No typos here, the account I am reffering to has the above mentioned username. I created my current account just to communicate with github support. Here's a screenshot from one of the repositories I was a collaborator for proof:
> > ![tmp_d49de136-6922-4d66-b8db-a347bcbc7e11](https://private-user-images.githubusercontent.com/187460821/385859327-3f8c731f-fccb-48ce-add4-b44ed4efbbd4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1MTc1MjAsIm5iZiI6MTczMTUxNzIyMCwicGF0aCI6Ii8xODc0NjA4MjEvMzg1ODU5MzI3LTNmOGM3MzFmLWZjY2ItNDhjZS1hZGQ0LWI0NGVkNGVmYmJkNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExM1QxNzAwMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NDZlY2M4ZGEwOTAwMTBmMzRkNDUxZGIzNjhmNTAwMzJiMTViYjlmNWE5ZTRkNGM2NzkxZmI3Y2JmYjFmMjY3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gAEvy8AhbBTFtFBH23UWg_GCXcymWhxVLKCtSSUQJrE)
> 
> Have you ever changed your username?

Nope, I have always used this same username.

**Comment by mastertanzeel** [comment]
Then the solution is to contact Github Support. They can tell you whats the issue exactly is.

**Reply by mouoent** [reply]
As I said in the post, I have already submitted a support ticket to GitHub support over a week ago and I still haven't received a reply. This is a very serious issue since I need my account for work and I am running out of options.

**Reply by mastertanzeel** [reply]
Good luck to you. I hope you'll get that back.

**Comment by Joao-Pedro-JP** [comment]
+1

**Comment by mecodeatlas** [comment]
Hi @mouoent!

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
